data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Function.java

```diff
@@ -59,7 +64,9 @@ public Function(String label, List<String> inColumnNameList, String outAlias) {
 	public void setOutNode(Node outNode) {this.outNode = outNode;}
 	public List<String> getInColumnNameList() {return inColumnNameList;}
 	public void setInColumnNameList(List<String> inColumnNameList) {this.inColumnNameList = inColumnNameList;}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)